mainWindow.py

```diff
@@ -13,7 +13,7 @@ class Ui_MainWindow(object):
     def setupUi(self, MainWindow):
         MainWindow.setObjectName("MainWindow")
         MainWindow.setWindowModality(QtCore.Qt.NonModal)
-        MainWindow.resize(202, 200)
+        MainWindow.resize(400, 400)
         MainWindow.setStyleSheet("")
         MainWindow.setProperty("WindowStaysOnTopHint", True)
         self.centralwidget = QtWidgets.QWidget(MainWindow)
```

Comment:
No changes should be commited in this file as it's generated automatically from the `.ui` file (which is used by QtDesigner). Make all the changes of style on the `hbc.py` file

Reply:
You can put this under the comment `# Custom editions on QMainWindow` on `hbc.py`, just like I did :)